packages/driver/cypress/e2e/commands/actions/click.cy.js

```diff
@@ -4103,10 +4103,10 @@ describe('mouse state', () => {
         const coordsChrome = {
           clientX: 492,
           clientY: 9,
-          layerX: 492,
-          layerY: 215,
```

Comment:
layerX and layerY properties are non-standard. We already had these mouse props commented out for Firefox because it was returning odd results. I'm removing the tests for these non-standard properties on all browsers. https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/layerX

>Non-standard: This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.